great_expectations/rule_based_profiler/parameter_builder/metric_parameter_builder.py

```diff
@@ -26,7 +27,7 @@ def __init__(
         self,
         parameter_name: str,
         metric_name: str,
-        metric_domain_kwargs: Optional[Union[str, dict]] = "$domain.domain_kwargs",
+        metric_domain_kwargs: Optional[Union[str, dict]] = None,
```

Comment:
What is the purpose of passing in `metric_domain_kwargs` as an argument in this case? Aren't we ignoring it here, and only getting it from the config?

Reply:
@talagluck That is the thing: most of the time, we are *NOT* ignoring it, but taking advantage of it!